.github/workflows/build-deploy.yml

```diff
@@ -33,6 +33,8 @@ jobs:
         echo HOST =${{ secrets.HOST }} >> .env
         echo REACT_APP_MATOMO_ID=${{ secrets.MATOMO_ID }} >> .env
         echo REACT_APP_MATOMO_ENDPOINT=${{ secrets.MATOMO_ENDPOINT }} >> .env
+        echo REACT_APP_SENTRY_DSN =${{ secrets.SENTRY_DSN }} >> .env
```

Comment:
I think it needs to be `secrets.REACT_APP_SENTRY_DSN`

Reply:
Updated both lines to read `secrets.REACT_APP_SENTRY_DSN` and `secrets.REACT_APP_SENTRY_SAMPLE_RATE` respectively.